docs_src/_includes/header.html

```diff
@@ -0,0 +1,43 @@
+<div id="header">
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+        <a href="{{ site.baseurl }}/"><img class="banner-img" src="{{site.baseurl}}/assets/img/logo.png"></a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav"
+                aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNav">
+            <ul class="navbar-nav ml-auto">
+
+                {% assign navButtons = "home|projects|publications|people|gallery|about" | split: "|" %}
```

Comment:
not fan of hard-coded navigation.

Reply:
I agree. I assumed the navigation bar is not gonna change frequently, so it is fine to be hard-coded. :)

Reply:
I think if we bring back the Minima theme, we can remove this file.